twoliter/src/tool-crates/buildsys/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
```

Comment:
Is there no way to combine these crates with the actual subcrates for the tool itself? Why not add the buildscript to the actual buildsys crate?

Reply:
I may not be understanding, do you mean add a sub-crate to `tools/buildsys` that does this, then pull it in from `twoliter`?

Or do you mean modify the existing `buildsys` crate? If that's what you mean, I'm pretty sure this would be a circular dependency -- We can't have buildsys requiring a pre-compiled version of itself in order to compile.